cmd/root.go

```diff
@@ -40,19 +40,19 @@ import (
 )
 
 var (
-	LogWithCommand           logrus.Entry
-	SubCommand               string
-	cfgFile                  string
-	databaseConfig           config.Database
-	diffBlockFromHeadOfChain int64
-	genConfig                config.Plugin
-	ipc                      string
-	maxUnexpectedErrors      int
-	recheckHeadersArg        bool
-	retryInterval            time.Duration
-	startingBlockNumber      int64
-	storageDiffsPath         string
-	storageDiffsSource       string
+	LogWithCommand              logrus.Entry
+	SubCommand                  string
+	cfgFile                     string
+	databaseConfig              config.Database
+	newDiffBlockFromHeadOfChain int64
```

Comment:
would it make sense to declare this with `unrecognizedDiffBlockFromHeadOfChain`?

Reply:
Yeah, good call. I had put it in the execute cmd file, but having it in root.go would be more consistent.